roles/testnode/vars/fedora_22.yml

```diff
@@ -70,28 +61,20 @@ packages:
   - blktrace
   - numpy
   - python-matplotlib
-  ###
-  # for qemu
-  - qemu-system-x86
-  - genisoimage
-  ###
-  # for apache and rgw
-  - httpd
-  - httpd-devel
-  - httpd-tools
-  - mod_ssl
-  - mod_fastcgi-2.4.7-1.ceph.fc20
-  ###
-  - libevent-devel
-  - python-devel
   # for json_xs to investigate JSON by hand
   - perl-JSON
   # for pretty-printing xml
   - perl-XML-Twig
   # for java bindings, hadoop, etc.
-  - java-1.7.0-openjdk-devel
-  - junit4
+  - java-1.8.0-openjdk-devel
+  - junit
   # for disk/etc monitoring
+  - nrpe
+  - nagios-plugins-all
   - smartmontools
   # for nfs
   - nfs-utils
+  # needed until we stop using yum ansible module
+  - yum
+  # python-pip is installed via roles/testnode/tasks/pip.yml on other rpm-based distros
+  - python-pip
```

Comment:
Because these packages get installed when the post-install trigger is called and cobbler runs the testnodes role on the host.